src/utilities/ci.jl

```diff
@@ -4,21 +4,59 @@ struct GitHubActions <: CIService
     username::String
     email::String
 end
+
+struct GitLabCI <: CIService
+    username::String
+    email::String
+end
```

Comment:
I can't comment around this for the `GitHubActions` :/ 
Note: This doesn't allow you to do `GitLabCI("username", "email")` though.

Not sure if that's functionality we want to keep or not /shrug

```suggestion
struct GitLabCI <: CIService
    username::String
    email::String
    
    function GitLabCI()
        return new("gitlab-ci[bot]", "gitlab-ci[bot]@gitlab.com")
    end
end
```